wordpress-mobile/WordPress-iOS
# Reader Improvements Detail: Address design review feedback

Fixes #15052

### To test:
1. Launch the app
2. Tap on Reader Detail
3. Make sure the bottom toolbar icons are all the same color in light and dark modes 
4. Tap on a topic
5. Make sure the back button says back
4. Tap back
5. Tap back again to go back to the feed
5. Make sure the Search icon is not black

This does not add any options to the more menu which will need to be addressed in a future task. 

### PR submission checklist:
